tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -1618,6 +1618,9 @@ public void listLocalesForImportOldVotes(User user, SurveyMain sm,
          */
         JSONObject oldVotesNull = null;
         localeCount.forEach((loc, count) -> {
+            if ("und".equals(loc)) {
```

Comment:
why?

Reply:
Without this, CLDRLocale.getInstance throws an exception for "und"

Given:
`private static final ULocale UNKNOWN_LOCALE = new ULocale("und");`
I don't know what it would mean to import old votes for UNKNOWN_LOCALE, so it seems better to skip it rather than throw an exception during import old votes.